### DISCUSSION TITLE
Assistance setting up DJI SDK Plugin for Android

### DISCUSSION THREAD

**Discussion by frankievx** [ionic-team/capacitor#3617] [Q&A]
I have this issue where the DJI SDK requires me to install these Helper functions to a custom application class. However, if I use the custom application class my webview doesn't display. If I remove the helper functions then I encounter other errors when attempting to register my DJI application.  Is there anyway I can extend the Application class with the webview if that makes sense? 

This seems similar to the situation I'm facing when integrating with Capacitor.
https://stackoverflow.com/questions/56262211/dji-sdk-implement-helper-install-in-android-dynamic-function

I'm new to android development so bear with me. 
<img width="762" alt="Screen Shot 2020-09-25 at 4 52 19 PM" src="https://user-images.githubusercontent.com/12982201/94314897-8a89b000-ff4f-11ea-84e3-b809ca9bf086.png">

**Comment by CodingTomato** [comment]
Any luck with your implementation?
